worker/gc/gc.go

```diff
@@ -90,3 +90,35 @@ func (g *GarbageCollector) DelPvPvcByServiceID(serviceGCReq model.ServiceGCTaskB
 		logrus.Warningf("service id: %s; delete a collection fo PVC: %v", serviceGCReq.ServiceID, err)
 	}
 }
+
+// DelKubernetesObjects deletes all kubernetes objects.
+func (g *GarbageCollector) DelKubernetesObjects(serviceGCReq model.ServiceGCTaskBody) {
+	deleteOpts := &metav1.DeleteOptions{}
+	listOpts := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("service_id=%s", serviceGCReq.ServiceID),
```

Comment:
LabelSelector should include `creator=Rainbond`?

Reply:
Added